src/actions/Entities/NameNoteActions.js

```diff
@@ -20,6 +20,8 @@ export const NAME_NOTE_ACTIONS = {
 };
 
 const validateData = (jsonSchema, data) => {
+  if (!jsonSchema) return true;
```

Comment:
the validator throws an error if the schema is null

Reply:
🤔 How does this even occur? Thought we had `{ surveySchema && <JSONForm/> }` everywhere